std_msgs/Header header
string mission_id
string mission_type
string status
float64 progress_percentage
string current_waypoint
string[] payload_status 